docs/get-started/deployment-and-usage.md

```diff
@@ -300,12 +300,13 @@ We have introduced the following Chart parameters.
 | `master.featureRulesController` | bool | null                                   | Specifies whether the controller for processing of NodeFeatureRule objects is enabled. If not set, controller will be enabled if `master.instance` is empty.
 | `master.replicaCount`       | integer | 1                                       | Number of desired pods. This is a pointer to distinguish between explicit zero and not specified                                         |
 | `master.podSecurityContext` | dict    | {}                                      | SecurityContext holds pod-level security attributes and common container settings                                                        |
+| `master.securityContext`    | dict    | {}                                      | Container [security settings](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod)|
```

Comment:
nit: The link refers to Pod security context. Could you point into **container** security, context instead?

Related suggestion: you could update `master.podSecurityContext` above similarly (to have a link) 😊